validator/src/main/java/io/avaje/validation/core/adapters/BasicAdapters.java

```diff
@@ -112,6 +113,37 @@ public boolean validate(Object value, ValidationRequest req, String propertyName
     }
   }
 
+  private static final class LengthAdapter implements ValidationAdapter<Object> {
```

Comment:
wait why do we need a separate adapter? can we not reuse the size adapter? since the logic is the exact same?

Reply:
> since the logic is the exact same?


To be exactly the same, `@Length` would apply to Collection, Map and Array. Currently in this PR `@Length` only applies to `CharSequence`. Hmm, thinking that probably doesn't matter or could be seen as a positive.

Reply:
Merging in by just re-using the SizeAdapter for now.